warehouse/templates/manage/roles.html

```diff
@@ -23,10 +23,10 @@ <h2>Collaborators</h2>
   <div class="callout-block">
     <p>There are two possible roles for collaborators:</p>
     <dl>
+      <dt>Maintainer</dt>
+      <dd>May upload releases for a project. Cannot add collaborators, delete files, delete releases or delete the project.</dd>
       <dt>Owner</dt>
       <dd>Owns a package, may add other collaborators for that package, and upload releases for a package.</dd>
```

Comment:
While you're here, could you update this line to indicate that owners _can_ delete releases and _can_ delete the project?